sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md

```diff
@@ -0,0 +1,20 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: DesktopVirtualization
+namespace: Azure.ResourceManager.DesktopVirtualization
+require: https://github.com/Azure/azure-rest-api-specs/blob/49af362e33d89967d7776fdd3a26d5462c9fbb59/specification/desktopvirtualization/resource-manager/readme.md
+tag: package-2021-07
+output-folder: Generated/
+clear-output-folder: true
+mgmt-debug:
+  show-request-path: true
```

Comment:
We no longer need this to show the request paths, now it shows the request paths by default
```suggestion
```